File: python/programmers/level 1/2016년.py
def solution(a, b):
    answer = ''
    months = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30]
    weeks = ['FRI','SAT','SUN','MON','TUE','WED','THU']
    days = 0
    for day in range(a):
        days += months[day]
    days += b
    days -= 1
    return weeks[days % 7]

File: java/src/main/java/ps_traning/programmers/level_1/_2016년.java
package ps_traning.programmers.level_1;

public class _2016년 {
    public String solution(int a, int b) {
        String answer = "";

        int[] months = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30};
        String[] weeks = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
        int days = b;
        for (int i = 1; i < a; i++) {
            days += months[i];
        }
        answer = weeks[days % 7];
        return answer;
    }
}
